cmd/process-agent/main.go

```diff
@@ -4,14 +4,19 @@ package main
 
 import (
 	"flag"
+	"github.com/DataDog/datadog-agent/cmd/process-agent/flags"
```

Comment:
Nit: separate stdlib imports from project imports

Reply:
Nice catch! Thanks for pointing this out! I've just updated the PR with the fix :)